src/test/java/io/jenkins/plugins/analysis/core/model/TabLabelProviderTest.java

```diff
@@ -0,0 +1,83 @@
+package io.jenkins.plugins.analysis.core.model;
+
+import org.junit.jupiter.api.Test;
+
+import edu.hm.hafner.analysis.Issue;
+import edu.hm.hafner.analysis.Report;
+
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
Please use AssertJ. That is checked by an architecture test: https://travis-ci.org/jenkinsci/warnings-ng-plugin/builds/514856410#L6587

Reply:
I changed the assertion library to assertj.